dev/fattest.simplicity/src/com/ibm/websphere/simplicity/ShrinkHelper.java

```diff
@@ -464,6 +464,22 @@ public static WebArchive defaultDropinApp(LibertyServer server, String appName,
         return app;
     }
 
+    /**
+     * Invokes {@link #buildDefaultApp(String, String...)}
+     * and then exports the resulting application to a Liberty server under the "dropins" directory
+     *
+     * @param server        The server to export the application to
+     * @param appname       The name of the application
+     * @param deployOptions options to configure how the application is deployed
+     * @param packages      A list of java packages to add to the application.
+     */
+    public static WebArchive defaultDropinApp(LibertyServer server, String appName, DeployOptions[] deployOptions, String... packages) throws Exception {
```

Comment:
I prefer to create a method `defaultDropinAppToServer` instead of this method if `SERVER_ONLY` is always passed in. @tevans78 is to investigate the usage and come up with a followup PR to fix this properly.